manifests/s/Sogou/SogouInput/14.3.0.9006/Sogou.SogouInput.installer.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.3.4 Dumplings Mod $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Sogou.SogouInput
+PackageVersion: 14.3.0.9006
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- sgreg
+FileExtensions:
+- scel
+- scpf
+- ssf
+ReleaseDate: 2024-03-18
+Installers:
+- Architecture: x86
+  InstallerUrl: https://ime.gtimg.com/pc/build/_sogou_pinyin_[Release]_14.3.0.9006_0.exe

```

Comment:
Hi @SpecterShell,

The `PackageUrl` leads to a slightly different URL - but it seems like a slightly different version number in the name. 

```
  InstallerUrl: https://ime-sec.gtimg.com/202403191007/ce4ed40503b739a1366258780417098d/pc/sogou_pinyin_guanwang_14.3a.exe
```

I would suggest it, except that the page doesn't allow access to download the file:

![image](https://github.com/microsoft/winget-pkgs/assets/114952565/f1d6f8d1-b21b-4cdf-ac16-56ebead4c319)

Is there a better `PacakgeUrl`?

Reply:
Actually there is no permanent link on their website and the link on the homepage will be updated every time you refresh the page.

The method I was using to obtain a permanent link no longer works starting from last month. This `PackageUrl` comes from their update API and should be the best one I've found.

Reply:
@stephengillie